util/pkg/vfs/s3fs.go

```diff
@@ -286,6 +288,9 @@ func (p *S3Path) getRequestACL(aclObj ACL) (*string, error) {
 }
 
 func (p *S3Path) WriteFile(ctx context.Context, data io.ReadSeeker, aclObj ACL) error {
+	ctx, span := tracer.Start(ctx, "S3Path::WriteFile")
```

Comment:
```suggestion
	ctx, span := tracer.Start(ctx, "S3Path::WriteFile", trace.WithAttributes(attribute.String("path", p.String())))
```

Reply:
Thanks!   I'm still trying to figure out when best to use attributes etc!